CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/domain/data/entity/UserEntity.kt

```diff
@@ -0,0 +1,59 @@
+package co.nimblehq.coroutine.domain.data.entity
+
+import co.nimblehq.coroutine.data.service.response.UserResponse
+
+data class UserEntity(
+    val id: Int?,
+    val name: String,
+    val username: String,
+    val email: String?,
+    val address: Address?,
+    val phone: String,
+    val website: String
+) {
+    data class Address(
+        val street: String,
+        val suite: String,
+        val city: String,
+        val zipCode: String,
+        val geo: Geo?
+    ) {
+        data class Geo(
+            val latitude: String,
+            val longitude: String
+        )
+    }
+}
+
+fun List<UserResponse>.toUsersEntity(): List<UserEntity> {
```

Comment:
@AVI5HEK  For `List<UserResponse>.toUserEntities()` It seems like we have more than one `Entity` for user but actually we have just only one so that Let's keep it like this and I think It more clear rather than `List<UserResponse>.toUsers()`  because we try to covert from response to entity 🙏

Reply:
In my opinion it should be:

- `.toUserEntities()`
- `.toEntities()`


Because `List<UserEntity>` is a list of user entities, right? 🙏